### PR TITLE
Leaderboards

### PR DESCRIPTION
**Pull request by Aj-Ong**

### CODE REVIEW

File: server/app.py [hunk 1]
```diff
@@ -266,50 +262,80 @@ def player_finished(data):
...
         return fetch_db_helper(all_users)
 
+    if sort_by == "gamesplayed":
+        all_users = DB.session.query(models.Users).order_by(
+            models.Users.gamesplayed.desc()).all()
+        return fetch_db_helper(all_users)
+
+    if sort_by == "gameswon":
+        all_users = DB.session.query(models.Users).order_by(
+            models.Users.gameswon.desc()).all()
```
Comment (yusefmustafa) [inline server/app.py:319]: Do we need to `return fetch_db_helper(all_users)` here as well?

File: server/app.py [hunk 2]
```diff
@@ -266,50 +262,80 @@ def player_finished(data):
     if players_finished_set == active_players_set:
         # We also include the winning player name in the 'gameComplete' message.
         winning_player = max(ROOMS[room]['activePlayers'], key=ROOMS[room]['activePlayers'].get)
-        print("THIS IS THE WINNING PLAYER:", winning_player)
-        print("THIS IS THE KEY:", ROOMS[room]['activePlayers'])
         update_db_gameswon(winning_player)
```
Comment (yusefmustafa) [inline server/app.py:265]: We need to make sure not to call `update_db_gameswon` if they are playing solo. Checking if `active_players_set` is 1 is enough.

**Review by yusefmustafa — APPROVED** [review]

File: server/app.py (hunk 1 above)
Comment (Aj-Ong) [reply server/app.py:319]: We dont need it as im returning it at the end of the function anyways, But I will add it to keep it unifrom with all the other if statements!

File: server/app.py (hunk 2 above)
Comment (Aj-Ong) [reply server/app.py:265]: Will do once we merge your changes to that function with mine and i will update on the next pull request

**Pull request merged**